frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -123,11 +124,11 @@ def frontier_stop(self):
 
     def add_seeds(self, seeds):
         for seed in seeds:
-            db_page, _ = self._get_or_create_db_page(seed)
+            self._create_db_page(seed, 0)
         self.session.commit()
 
     def get_next_requests(self, max_next_requests, **kwargs):
-        query = self.page_model.query(self.session)
+        query = self.page_model.query(self.session).with_lockmode('update')
```

Comment:
That should cause slowdown in general single process state.Would be nice to hide it with option, probably.